### PR TITLE
added Susan's bio

### PR DESCRIPTION
**Pull request by lesleychard**

### CODE REVIEW

File: modules/home/HomeOurTeam.tsx [hunk 1]
```diff
@@ -117,6 +117,21 @@ const TEAM_MEMBERS = [
...
+    name: 'Susan Kelly',
+    position: 'High School Talent Lead',
+    company: 'TechNL',
+    imageUrl: 'assets/images/bios/susan-kelly.jpg',
+    profile: (
+      <Typography>
+        Susan Kelly is an experienced educator and passionate community connector who is currently serving as the High School Talent Lead at techNL.
+        She holds a B. Ed and M. Ed (Leadership Studies) and has professional experience in the education and community sectors.
+        Susan has worked as a classroom teacher in various communities in the province as well as in the community sector with youth and social programming.
+        She is passionate about inspiring students to discover a passion for tech and envision meaningful careers right here in Newfoundland and Labrador
```
Comment (jillBreau) [inline modules/home/HomeOurTeam.tsx:131]:
```suggestion
        She is passionate about inspiring students to discover a passion for tech and envision meaningful careers right here in Newfoundland and Labrador.
```

**Review by jillBreau — CHANGES_REQUESTED** [review]
Added one requested change.

Also, I know you just changed the `company` to "TechNL", but they do say "techNL" on their website and social media. I think either way, we should be consistent with the capitalization across the `company` and the mention in `profile`.

File: modules/home/HomeOurTeam.tsx (hunk 1 above)
Comment (lesleychard) [reply modules/home/HomeOurTeam.tsx:131]: Ah, I wasn't sure. I'll remove the capitalization again 😅

Comment (jillBreau) [reply modules/home/HomeOurTeam.tsx]: Yeh it's kind of hard to decide 😬

**Review by jillBreau — APPROVED** [review]
Looks great!

**Pull request merged**